recipe/meta.yaml

```diff
@@ -28,10 +26,11 @@ test:
 
 about:
   home: http://code.google.com/p/google-diff-match-patch/
-  license: Apache 2.0
-  license_file: LICENCE
-  summary: 'The Diff Match and Patch libraries offer robust algorithms to perform the operations required for synchronizing plain text.'
+  license: Apache-2.0
+  license_file: LICENSE
+  summary: 'Diff Match Patch is a high-performance library in multiple languages that manipulates plain text'
```

Comment:
That was too long and the one on their GH page is virtually the same but shorter.